### ISSUE TITLE
Smart taskbar not working on latest windows 11 version

### ISSUE THREAD

**Issue by AdwaitVan** [ChanpleCai/SmartTaskbar#136]
I use smart taskbar and it has suddenly crashed after an windows update meaning after that update, whenever doubled clicked on icon at taskbar it didn't worked. I tried reinstalling and checking everything but its not happening. also when i clicked on auto after right clicking on the icon , the taskbar starting jittering up and down and kept doing it. it was definitely a glitch. so please make the changes i really need this app properly as i use it very much.

**Comment by juergen61** [comment]
i confirm the same problem. It appears  since win11 update (KB5022913) at the beginning of March.
Microsoft added functions for taskbar, search bar and s.o. 
smart taskbar did not work anymore. after doublecliclk the taskbar starting jittering.

today i tested again.  no jitterig, but the opend  (maximized) program-windows tried to resize to use area of the taskbar, but taskbar did not disappear, so in the same second they go back in old position when taskbar is visible it makes just a twitch.
taskbar did not move down.

Hope this helps to solve the problem

**Comment by AdwaitVan** [comment]
I am sorry actually .....I didnt understand..So its not jittering for you now ? what did you do? Yes it started jittering after they rolled out additional features for taskbar,search bar etc. please help me out here

**Comment by juergen61** [comment]
i can not say when behave changes. I installed an alternative tool, "hide taskbar" . Maybe that changes somthing. Or a new microsoft update make the change.
But smart taskbar did still not work, only jittering disappear.
Hide taskbar has a bug so it is not the best alternative.

**Comment by juergen61** [comment]
hi i tested it again, jittering is still there. when i checked it 3 days ago, i only startet smart taskbar and try doubleclick to hide. Today i changed between different opend an maximzed program-windows to hide taskbar, and jittering is still there. sorry for incorrect info.

**Comment by AdwaitVan** [comment]
yes i told u there its jittering. then what to do like this problem was there before too for previous verisons. how to let know developer to fix the bug?

**Comment by juergen61** [comment]
Hi, the developer read this issues, if you look at older issues, you see he answers , sometimes.
i wonder that we both are the only user with this problem.

**Comment by AdwaitVan** [comment]
I dont think so there must others as well but i use this app literally daily so i need to get it fixed. thats y i insist. how to make my above comment as bug issue any idea?

**Comment by juergen61** [comment]
the developer read this, but you can contact him via mail. click on his name on top left to get mail adress.
until the program is fixed, i use power-shell commads to hide and show the taskbar. works fine for me.

**Comment by AdwaitVan** [comment]
how can you help me out there please

**Comment by juergen61** [comment]
[Taskbar.zip](https://github.com/ChanpleCai/SmartTaskbar/files/11174404/Taskbar.zip)

Hi
unpack the zip. hide.cmd and show.cmd are windows powershell commands. if you doubleclick on hide, taskbar will hide , click on show it will show taskbar.
put the cmd-files  in a new directory. then make links for both.
Put the links on your desktop. go to properties of the links and change propertie "run:" to minimized. this hides the powershell-window that opens when clicking on the command files. You start hide und show only with the links on desktop. Also you can add hotkeys to the links, than you can hide and show without click, just with the hotkeys.
if you need help, ask me.

**Comment by AdwaitVan** [comment]
Thank you very much for your this.

**Comment by sameerasw** [comment]
+1

Can confirm that Smart Taskbar is broken (for me , it's since the launch of the new rebooted Windows 11 Insider Dev channel)

**Comment by AdwaitVan** [comment]
> [Taskbar.zip](https://github.com/ChanpleCai/SmartTaskbar/files/11174404/Taskbar.zip)
> 
> Hi unpack the zip. hide.cmd and show.cmd are windows powershell commands. if you doubleclick on hide, taskbar will hide , click on show it will show taskbar. put the cmd-files in a new directory. then make links for both. Put the links on your desktop. go to properties of the links and change propertie "run:" to minimized. this hides the powershell-window that opens when clicking on the command files. You start hide und show only with the links on desktop. Also you can add hotkeys to the links, than you can hide and show without click, just with the hotkeys. if you need help, ask me.

How to put cmd files in new directory and make them link together? and actually can u contact me on discord to easy communication. username is vanmali.ak.johnny#8675

**Comment by AdwaitVan** [comment]
I have emailed the developer regarding the issue lets see if they reply on it. Hope they reply.

**Comment by Ymigmli** [comment]
https://github.com/TranslucentTB/TranslucentTB/issues/537

和TranslucentTB问题原因一样，可以用于参照临时解决本问题
1. 下载[ViVeTool](https://github.com/thebookisclosed/ViVe)
2. 管理员权限运行`ViVeTool.exe /disable /id:26008830 /variant:2`
3. 在D盘根目录新建一个re.txt，里面放入`ViVeTool.exe /enable /id:26008830 /variant:2`
4. 当旧任务栏在新版本中删除，更新进入系统后可能会黑屏，ctrl+alt+del调出任务管理器-运行新任务-勾选以系统管理员权限创建此任务-输入powershell回车-在powershell中输入`d:`回车，输入`cat .\re.txt`并回车-输入回显内容并回车，重启即可。

**Comment by Himaannshuu** [comment]
Any Updates on this issue??

**Comment by juergen61** [comment]
No, i use power shell commands until Smart taskbar works again

**Comment by isargb** [comment]
Try to use the v1.4.3. I have Windows 11 Pro 22H2 build 22621.2283 and using v1.4.3. solved my issue.

**Comment by Aronjlove** [comment]
> Try to use the v1.4.3. I have Windows 11 Pro 22H2 build 22621.2283 and using v1.4.3. solved my issue.

In case anyone comes looking, v1.4.3 is working on Windows 11 Pro 24H2 build 26100.1876, and Windows 11 Enterprise 23H2 build 22631.4169.
